package.json

```diff
@@ -6,6 +6,7 @@
     "uglify-js": "2.4.24"
   },
   "devDependencies": {
-    "jshint": "^2.7.0"
+    "jshint": "^2.7.0",
+    "edx-custom-a11y-rules": "edx/edx-custom-a11y-rules"
```

Comment:
I think this is installing from github,  https://docs.npmjs.com/files/package.json#github-urls.  That seemed like the simplest solution right now, but we could publish the repo to npm if we wanted to.